src/components/AppHeader/AppHeader.jsx

```diff
@@ -1,6 +1,15 @@
-// import Logo from "@/assets/spa-logo.png";
-// import "./AppHeader.scss";
-// const AppHeader = () => (
-// what goes in here? there is no content in the AppHeader component
-// );
-// export default AppHeader;
+import Logo from "@/assets/spa-logo.png";
```

Comment:
I think it would be better to use the hotel-logo.png here.

Reply:
I have changed it.